tests/integration/workspace_access/test_groups.py

```diff
@@ -207,7 +212,7 @@ def test_group_matching_names(ws, sql_backend, inventory_schema, make_ucx_group,
 
 
 # average runtime is 100 seconds
-@retried(on=[NotFound], timeout=timedelta(minutes=3))
+@retried(on=[NotFound], timeout=timedelta(minutes=15))
```

Comment:
Don't modify this test, create a new one just for redash queries. It should be in test_redash.py, same as for generic/scim/etc

Mindlessly modifying unrelated tests has previously led them being unmaintainable and i had to spend days rewriting them

Reply:
will do

Reply:
done